### PR TITLE
Classroom

### PR DESCRIPTION
**Pull request by matthewGallagher**

### CODE REVIEW

File: app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -42,14 +54,27 @@
...
         <item name="colorPrimaryDark">@color/lockerPrimaryDark</item>
         <item name="android:colorBackground">@color/lockerBackground</item>
         <item name="android:buttonStyle">@style/ButtonTheme</item>
     </style>
+<<<<<<< HEAD
+
+    <!-- Quiz Theme. -->
+
+=======
+>>>>>>> 7fd079e792bc2220c9bb7c5f1fe7f4839219a2f3
```
Comment (mderb4) [inline app/src/main/res/values/styles.xml]: Merge conflict needs to be updated and repushed.

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -6,6 +6,7 @@
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
+        android:theme="@style/Theme.AppCompat.Light"
```
Comment (sklei2) [inline app/src/main/AndroidManifest.xml]: Not necessary. We're doing themes on an activity level. Not app level.

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -16,12 +17,29 @@
             </intent-filter>
         </activity>
         <activity
+
+            android:name="swen_anigains.mathematicfanatic.ClassroomActivity"
```
Comment (sklei2) [inline app/src/main/AndroidManifest.xml]: No need for the "swen_anigans.mathematicfanatic"

File: app/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -16,12 +17,29 @@
...
+            android:label="@string/title_activity_classroom"
+            android:parentActivityName=".MainActivity"
```
Comment (sklei2) [inline app/src/main/AndroidManifest.xml]: The parent of the classroom is the main activity? This should probably be the locker.

File: app/src/main/AndroidManifest.xml [hunk 5]
```diff
@@ -16,12 +17,29 @@
...
+            android:theme="@style/AppTheme.NoActionBar" />
+        <activity
             android:name=".GuardianSettings"
             android:parentActivityName=".MainActivity">
 
         </activity>
         <activity android:name=".StudentSettings"/>
         <activity
+            android:name=".ClassroomActivity"
```
Comment (sklei2) [inline app/src/main/AndroidManifest.xml]: Is this a repeat of the above activity? remove the activity above.

File: app/src/main/AndroidManifest.xml [hunk 6]
```diff
@@ -16,12 +17,29 @@
...
+            android:theme="@style/ClssroomTheme"
+            android:parentActivityName=".StudentLocker">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
```
Comment (sklei2) [inline app/src/main/AndroidManifest.xml]: Remove the intent-filter. that's only for the main activity.

File: app/src/main/res/values/colors.xml [hunk 7]
```diff
@@ -13,7 +13,7 @@
 
     <!-- Guardian Settings/PIN -->
     <color name="guardianPrimary">#607D8B</color>
-    <color name="guardianPrimaryDark">#263238</color>
+    <color name="guardianPrimaryDark">#000000</color>
```
Comment (sklei2) [inline app/src/main/res/values/colors.xml]: Don't change the guardian colors

File: app/src/test/java/swen_anigains/mathematicfanatic/ExampleUnitTest.java [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class ExampleUnitTest {
+    @Test
+    public void addition_isCorrect() throws Exception {
+        assertEquals(4, 2 + 2);
+    }
+}
```
Comment (sklei2) [inline app/src/test/java/swen_anigains/mathematicfanatic/ExampleUnitTest.java]: Was this added by accident?

File: app/src/androidTest/java/swen_anigains/mathematicfanatic/ExampleInstrumentedTest.java [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+@RunWith(AndroidJUnit4.class)
+public class ExampleInstrumentedTest {
+    @Test
+    public void useAppContext() throws Exception {
+        // Context of the app under test.
+        Context appContext = InstrumentationRegistry.getTargetContext();
+
+        assertEquals("swen_anigains.mathematicfanatic", appContext.getPackageName());
+    }
+}
```
Comment (sklei2) [inline app/src/androidTest/java/swen_anigains/mathematicfanatic/ExampleInstrumentedTest.java]: Was this added by accident?

**Pull request merged**